src/lua/init.lua

```diff
@@ -29,6 +29,121 @@ local LUA_TEMPLATES = { '?.lua', '?/init.lua' }
 local ROCKS_LIB_TEMPLATES = { ROCKS_LIB_PATH .. '/?.'..soext }
 local ROCKS_LUA_TEMPLATES = { ROCKS_LUA_PATH .. '/?.lua', ROCKS_LUA_PATH .. '/?/init.lua' }
 
+-- A wrapper with a unique name, used in the call stack traversing.
+local function wrapper_e6c2bafc7d0611ec8008db177bebd1df()
```

Comment:
Nit: This name must surely be unique, but I would use something more human-readable and less mysterious :-)
For example, `__tarantool__internal__require__wrapper__` or something like that.

Reply:
Done.